AutocompleteClientTests/Logic/Query/Builder/TrackConversionRequestBuilderTests.swift

```diff
@@ -83,6 +83,14 @@ class TrackConversionRequestBuilderTests: XCTestCase {
         XCTAssertTrue(request.url!.absoluteString.contains("_dt=\(requestDate)"), "URL should contain the request date")
     }
 
+    func testTrackConversionBuilder_withNoSectionSpecified_hasNoSectionName() {
+        let tracker = CIOConversionTrackData(searchTerm: searchTerm)
+        builder = TrackConversionRequestBuilder(tracker: tracker, autocompleteKey: testACKey)
+        let request = builder.getRequest()
+        
+        XCTAssertTrue(!request.url!.absoluteString.contains("\(Constants.TrackAutocomplete.autocompleteSection)="), "URL shouldn't contain the default autocomplete section if the section name is not passed.")
```

Comment:
This test seems not right... wouldn't the url string check for `section_name=`?

Reply:
There's a NOT operator before the contains call. One could very well remove it and replace with XCTAssertFalse